tests/lib/external/snapd-testing-tools/tests/os.query/task.yaml

```diff
@@ -143,12 +150,7 @@ execute: |
             os.query is-classic
             ! os.query is-core
             ;;
-        centos-7-64)            
-            os.query is-centos
-            os.query is-centos 7
             os.query is-classic
```

Comment:
Was this line supposed to be deleted as well?

Reply:
Or perhaps moved to below `centos-9-64)` ?

Reply:
yes, it was moved to centos-9, this is how is it displayed on github

Reply:
I just cloned the repo to check, and it looks like it is actually above the `centos-9` case:

```
        amazon-linux-2023-64)
            os.query is-amazon-linux
            os.query is-amazon-linux 2023
            ! os.query is-amazon-linux 2
            os.query is-classic
            ! os.query is-core
            ;;
            os.query is-classic
        centos-9-64)            
            os.query is-centos 9
            os.query is-centos
            ! os.query is-core
            ;;
        opensuse-15.5-64)
```

Reply:
I see something different:
https://github.com/canonical/snapd-testing-tools/blob/main/tests/os.query/task.yaml#L153

Reply:
the problem was caused by the merge tool, I just fixed that in snapd pr.

Reply:
thanks for catching it